.github/workflows/pull-compliance.yml

```diff
@@ -0,0 +1,145 @@
+name: "Pull: Compliance Tests"
+
+on: [pull_request]
+
+jobs:
+  lint_basic:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: setup go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.20'
+      - name: deps-backend
+        run: make deps-backend deps-tools
+      - name: lint backend
+        run: make lint-backend
+        env:
+          GOPROXY: https://goproxy.io # proxy.golang.org is blocked in China, this proxy is not
+          GOSUMDB: sum.golang.org
+          TAGS: bindata sqlite sqlite_unlock_notify
+  lint_windows:
+    runs-on: ubuntu-latest
```

Comment:
Run on `windows`? 😆

Reply:
> Not sure if gitea actions can

mostly-ish it can, but it's host mode only, and I don't want to have to deal with a bunch of Windows VMs. So to keep this portable to Gitea Actions I have it running on linux.

Reply:
Fine with me, it was never running on Windows anyways.